src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/model/Prisoner.kt

```diff
@@ -13,77 +14,82 @@ import java.time.LocalDate
 
 @Entity
 @Table(name = "PRISONERS", schema = "HPA")
-open class Prisoner {
+data class Prisoner(
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   @Column(name = "PK_PRISONER", nullable = false)
-  open var id: Int? = null
+  @JsonIgnore
+  val id: Int? = null,
```

Comment:
Should an id be nullable? I imagine there are `!!`s everywhere but it will never actually be null.

Reply:
Also a little surprised it's not being returned, I guess it's not being used as the unique key then?